lib/external.php

```diff
@@ -26,7 +26,8 @@
 class External {
 
 	public static function getSites() {
-		if (($sites = json_decode(\OCP\Config::getAppValue("external", "sites", ''))) != null) {
+		if (($sites = \json_decode(\OC::$server->getConfig()
+				->getAppValue("external", "sites", ''))) != null) {
```

Comment:
it would be better to split the assignment and the condition. I read this statement in a wrong way at first.